### PR TITLE
[ADD] basics to get working on Centos 7

### PR DESCRIPTION
**Pull request by bigUNO**
@kbrebanov Just a quick hack to get the RedHat/CentOS parts working. 🕉

### CODE REVIEW

**Comment by bigUNO** [comment]
I found some errors that need fixing. I'll re-open when I've fully tested on CentOS.

**Comment by bigUNO** [comment]
@kbrebanov Let's try this again. 😄

File: README.md [hunk 1]
```diff
@@ -65,6 +65,32 @@ Install rsyslog and configure logging options for HAProxy
...
+        lines:
+          - "ruleset(name=\"ruleset_remote\") {"
+          - "   action(type=\"omfile\" template=\"outputformat_raw\" dynaFile=\"dynafile_undefined\")"
+          - "   stop"
+          - "}"
+
+      - name: custom-listeners
+        priority: 99
+        lines:
+          - "input(type=\"imudp\" port=\"514\" ruleset=\"ruleset_remote\")"
```
Comment (lhoss) [inline README.md:90]: easy avoid escaping by using diff quotes:
```
- 'input(type="imudp"  .... '
```

**Comment by lhoss** [comment]
I'm planing to use @kbrebanov 's role for our rsyslog deployment, then found @bigUNO useful additions (in his fork), with the support for later rh/centos  but also adding a bunch of tests👍 
( though for now we just require support for Ubuntu 14.4, 16.4 )
So I will try this latest state for some test deploys on those distro versions, report any found issues (and fallback to the orig role, if blocked)

ps: I always recommend any ansible devOps to check out https://github.com/metacloud/molecule  for leaner, more declarative & boilerplate-less testing

**Comment by kbrebanov** [comment]
This Ansible role is no longer being maintained.

**Pull request closed without merging**